internal/admin/authorizedapps/view.go

```diff
@@ -35,6 +37,23 @@ func NewView(c *admin.Config, env *serverenv.ServerEnv) admin.Controller {
 	return &viewController{config: c, env: env}
 }
 
+func addHealthAuthorityInfo(ctx context.Context, haDB *verdb.HealthAuthorityDB, app *model.AuthorizedApp, m admin.TemplateMap) error {
+	// Load the health authorities.
+	healthAuthorities, err := haDB.ListAllHealthAuthoritiesWithoutKeys(ctx)
+	if err != nil {
+		return fmt.Errorf("Error loading health authorities: %w", err)
+	}
+
+	usedAuthorities := make(map[int64]bool)
```

Comment:
struct{} instead of bool?

Reply:
has to be bool for the html templates to work correctly (tried struct)